open_spiel/python/pybind11/pyspiel.cc

```diff
@@ -296,6 +296,9 @@ PYBIND11_MODULE(pyspiel, m) {
            (std::vector<int>(State::*)(int) const) & State::LegalActionsMask)
       .def("legal_actions_mask",
            (std::vector<int>(State::*)(void) const) & State::LegalActionsMask)
+      .def("legal_chance_outcomes",
+           (std::vector<open_spiel::Action>(State::*)(void) const) &
```

Comment:
This cast is only needed in the case that there's an overloaded method name. It can be omitted here.

Reply:
Thanks! I have updated the PR